src/Plugin/pathauto/AliasType/TaxonomyTermAliasType.php

```diff
@@ -43,4 +43,16 @@ public function getSourcePrefix() {
     return 'taxonomy/term/';
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function applies($object) {
+    if (parent::applies($object)) {
+      /** @var \Drupal\taxonomy\TermInterface $object */
+      $config_forum = $this->configFactory->get('forum.settings');
+      return $object->getVocabularyId() != $config_forum->get('vocabulary');
```

Comment:
Interesting. a generic term implementation (this class should go away in favor of the derivative, right?) shouldn't have to know about forum.module?

Maybe it's up to the site builder to make sure that forum has a higher weight, if used? I have no idea if ForumAliasType currently works btw, I don't think we have any test coverage.